packages/office-ui-fabric-react/src/components/pickers/BasePicker.tsx

```diff
@@ -156,7 +156,7 @@ export class BasePicker<T, P extends IBasePickerProps<T>> extends BaseComponent<
           <SelectionZone selection={ this.selection } selectionMode={ SelectionMode.multiple }>
             <div className={ css('ms-BasePicker-text', styles.pickerText) } role={ 'list' }>
               { this.renderItems() }
-              <BaseAutoFill
+              { this.okToAddMore() && (<BaseAutoFill
                 { ...inputProps as any }
```

Comment:
One thing I'm not quite sure about is where focus will wind up. Normally when you select something, focus is given back to the input, in this case the input won't exist any more. I think focus and selection should be given to the last item selected in the list. I think you'll need to put the check in componentwillupdate